docs/faq.rst

```diff
@@ -319,6 +319,24 @@ I'm scraping a XML document and my XPath selector doesn't return any items
 
 You may need to remove namespaces. See :ref:`removing-namespaces`.
 
+How can I split an item into multiple items in an item pipeline?
+----------------------------------------------------------------
+
+:ref:`Item pipelines <topics-item-pipeline>` cannot yield multiple items per
+input item. :ref:`Create a spider middleware <custom-spider-middleware>`
+instead, and use its
+:meth:`~scrapy.spidermiddlewares.SpiderMiddleware.process_spider_output`
+method for this puspose. For example::
+
+    class MultiplyItemsMiddleware:
+
+        def process_spider_output(self, response, result, spider):
+            for item in result:
+                if isinstance(item, scrapy.Item):
```

Comment:
I think this is slightly more precise:
```suggestion
                if isinstance(item, (scrapy.item.BaseItem, dict)):
```
(it'd be good to add necessary imports as well)